sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -0,0 +1,87 @@
+import { Injectable, Inject } from "@nestjs/common";
+import { RecordDataModelService } from "../../database/data.model.service";
+import { Institution, User } from "../../database/entities";
+import { Connection } from "typeorm";
+import { UserInfo } from "../../types";
+import { CreateInstitutionDto } from "../../route-controllers/institution/models/institution.dto";
+import { LoggerService } from "../../logger/logger.service";
+import { BCeIDService } from "../bceid/bceid.service";
+import { InjectLogger } from "../../common";
+
+@Injectable()
+export class InstitutionService extends RecordDataModelService<Institution> {
+  @InjectLogger()
+  logger: LoggerService;
+  @Inject()
```

Comment:
Can you please inject this in the constructor as the other ones? I think that the logger would be the only exception.

Reply:
Make Sense. Done